File: jewelry-admin/merge.py
import os

with open("pages/invoices.js", "r", encoding="utf-8") as f:
    text = f.read()

text = text.replace("Invoice", "Memo")
text = text.replace("invoice", "memo")
text = text.replace("INVOICE", "MEMO")
text = text.replace("Invoices", "Memos")
text = text.replace("invoices", "memos")

with open("pages/memos.js", "w", encoding="utf-8") as f:
    f.write(text)

print("SUCCESSFULLY MERGED")

File: jewelry-admin/merge.js
const fs = require('fs');
try {
  let inv = fs.readFileSync('pages/invoices.js', 'utf8');
  let memo = inv.replace(/Invoice/g, 'Memo').replace(/invoice/g, 'memo');
  memo = memo.replace(/Invoices/g, 'Memos');
  memo = memo.replace(/invoices/g, 'memos');
  fs.writeFileSync('pages/memos.js', memo, 'utf8');
  console.log('Successfully wrote memos.js');
} catch(e) {
  console.error(e);
}
